lxd/response/response.go

```diff
@@ -277,22 +277,31 @@ func (r *errorResponse) Render(w http.ResponseWriter) error {
 
 // FileResponseEntry represents a file response entry.
 type FileResponseEntry struct {
+	// Required.
 	Identifier string
-	Path       string
 	Filename   string
-	Buffer     []byte /* either a path or a buffer must be provided */
+
+	// Read from a filesystem path.
+	Path string
+
+	// Read from a file.
+	File       io.ReadSeeker
+	FileSize   int64
+	FileModify time.Time
```

Comment:
Is this `Modify` rather than `Modified` to match another structure field or convention?

Reply:
Sent